hologram_dashboard/proto++/proto_value.cc

```diff
@@ -0,0 +1,52 @@
+/*
+*   Copyright 2020 Google LLC
+*
+*   Licensed under the Apache License, Version 2.0 (the "License");
+*   you may not use this file except in compliance with the License.
+*   You may obtain a copy of the License at
+*
+*       https://www.apache.org/licenses/LICENSE-2.0
+*
+*   Unless required by applicable law or agreed to in writing, software
+*   distributed under the License is distributed on an "AS IS" BASIS,
+*   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+*   See the License for the specific language governing permissions and
+*   limitations under the License.
+*/
+
+#include "proto_value.h"
+
+namespace wireless_android_play_analytics {
+
+std::unique_ptr<ProtoValue> ProtoValue::Create(absl::string_view text_proto, 
```

Comment:
I really don't think you need unique_ptr or shared_ptr here.

Try this

https://screenshot.googleplex.com/AZqx8Czq0Tc